content/docs/command-reference/exp/run.md

```diff
@@ -68,37 +68,29 @@ committing them to the Git repo. Unnecessary ones can be removed with
 
 ## Checkpoints
 
-To track successive steps in a longer <abbr>experiment</abbr>, you can register
-checkpoints with DVC during your code or script runtime (similar to a logger).
+To track successive steps in a longer or deeper <abbr>experiment</abbr>, you can
+register checkpoints from your code (similar to a logger). Each `dvc exp run`
```

Comment:
I don't really understand the `similar to a logger` note here? Checkpoints don't really have anything to do with logging

Reply:
Good Q @pmrowla (and sorry for the delay). I didn't change that in this PR BTW.

Aren't checkpoints a way to log experiment steps though? We even have a logger library that registers checkpoints (DVCLive).

Open for suggestions though. Just remove the parenthesis?

Reply:
I would just remove it

Reply:
OK!
```suggestion
register checkpoints from your code. Each `dvc exp run`
```

Reply:
Done!